### PR TITLE
divik level

### PR DESCRIPTION
**Pull request by Sebaxp95**

### CODE REVIEW

File: src/Spectre.AngularClient/src/app/divik-results/divik/divik.component.html [hunk 1]
```diff
@@ -3,9 +3,19 @@
   {{configDescription}}
 </div>
 
+<md-form-field *ngIf="divikConfig">
+  <input mdInput
+         type='number'
+         min="0"
+         max={{divikConfig.Level}}
+         (change)="changeLevel($event.target.value)"/>
+  <md-hint align="center">Max value: divikConfig.Level</md-hint>
```
Comment (gmrukwa) [inline src/Spectre.AngularClient/src/app/divik-results/divik/divik.component.html]: This is not necessarily true. If the experiment has been ran with `useLevels: false`, backend should estimate tree depth and exhibit it to the frontend.

File: src/Spectre.AngularClient/src/app/divik-results/divik/divik.component.ts [hunk 2]
```diff
@@ -65,6 +67,15 @@ export class DivikComponent implements OnInit {
     return description;
   }
 
+  changeLevel(value) {
+    console.log('change ' + value);
+    if (value > 0 && value <= this.divikConfig.Level) {
+      this.divikService
+        .get(1, 1, value)
```
Comment (gmrukwa) [inline src/Spectre.AngularClient/src/app/divik-results/divik/divik.component.ts]: We should use known `preparationId`, shouldn't we? It has been passed through the router.

File: src/Spectre.AngularClient/src/app/divik-results/divik/divik.component.html (hunk 1 above)
Comment (Sebaxp95) [reply src/Spectre.AngularClient/src/app/divik-results/divik/divik.component.html]: So if we haven't had this information yet it could be another task

**Comment by gmrukwa** [comment]
[Enable DiviK depth selection](https://trello.com/c/g5Q58WR8/211-enable-divik-depth-selection)

**Pull request merged**